internal/engine/eval/vulncheck/review.go

```diff
@@ -305,52 +256,93 @@ func (ra *reviewPrHandler) setStatus() {
 	ra.logger.Debug().Str("status", *ra.status).Msg("will set review status")
 }
 
-func (ra *reviewPrHandler) findPreviousReview(ctx context.Context) error {
-	reviews, err := ra.cli.ListReviews(ctx, ra.pr.RepoOwner, ra.pr.RepoName, int(ra.pr.Number), nil)
+func (ra *reviewPrHandler) findPreviousStatusComment(ctx context.Context) error {
```

Comment:
It would be nice if this function didn't pass the found review as a struct field, but again, I'm not going to block this review on existing code styles.